boards/lilygo/ttgo_lora32/doc/index.rst

```diff
@@ -0,0 +1,241 @@
+.. _SX127x: https://www.semtech.com/products/wireless-rf/lora-connect/sx1276#documentation
+.. _ESP32-PICO-D4: https://www.espressif.com/sites/default/files/documentation/esp32-pico-d4_datasheet_en.pdf
+.. _SSD1306: https://cdn-shop.adafruit.com/datasheets/SSD1306.pdf
+
++------------------+-------------------------------------------------------------------------+
+| Key Component    | Description                                                             |
++==================+=========================================================================+
+| ESP32-PICO-D4    | This `ESP32-PICO-D4`_ module provides complete Wi-Fi and Bluetooth      |
+|                  | functionalities and integrates a 4-MB SPI flash.                        |
++------------------+-------------------------------------------------------------------------+
+| Diagnostic LED   | One user LED connected to the GPIO pin.                                 |
++------------------+-------------------------------------------------------------------------+
+| USB Port         | USB interface. Power supply for the board as well as the                |
+|                  | serial communication interface between a computer and the board.        |
+|                  | Micro-USB type connector.                                               |
++------------------+-------------------------------------------------------------------------+
+| Power Switch     | Sliding power switch.                                                   |
++------------------+-------------------------------------------------------------------------+
+| LCD screen       | Built-in OLED display \(`SSD1306`_, 0.96", 128x64 px\) controlled       |
+|                  | by I2C interface                                                        |
++------------------+-------------------------------------------------------------------------+
+| SX1276/SX1278    | LoRa radio frontend chip, connected via SPI.                            |
+|                  | Use SX1276 for 433MHz and SX1276 for 868/915/923MHz.                    |
++------------------+-------------------------------------------------------------------------+
+| TF card slot     | TF card slot wired to the SD interface of the MCU.                      |
++------------------+-------------------------------------------------------------------------+
+
+
+Start Application Development
+*****************************
+
+Before powering up your Lilygo ttgo LoRa32, please make sure that the board is in good
+condition with no obvious signs of damage.
+
+System requirements
+===================
+
+Prerequisites
+-------------
+
+Espressif HAL requires WiFi and Bluetooth binary blobs in order work. Run the command
+below to retrieve those files.
+
+.. code-block:: console
+
+   west blobs fetch hal_espressif
+
+.. note::
+
+   It is recommended running the command above after :file:`west update`.
+
+Building & Flashing
+*******************
+
+ESP-IDF bootloader
```

Comment:
@lfelten Sorry to report this again, but instead of squashing your second commit with doc changes, you seem to remove it. So the docs need to be fixed again.

Reply:
Hi Marek,
thanks for reviewing!
I fixed it for PR #70458 (Lilygo TTGO T8C3) and this PR #68179 is for the Lilygo TTGO LoRa32.
Since you're looking into this one too, I applied equivalent changes and force pushed the changes just now.

Reply:
@lfelten ah, you are right :)  Thanks for pointing this!